### PR TITLE
Fix markdown syntax

### PR DESCRIPTION
**Pull request by lipkau**
## PR Summary

* Added rule for markdown linter
  + this makes markdown linter ignore duplicate headers in the same document
* Fixed styling of community snippets
  + have all indentation to look alike
  + have code of snippets to use `\t`

## PR Checklist

Note: Tick the boxes below that apply to this pull request by putting an `x` between the square brackets.
Please mark anything not applicable to this PR `NA`.

- [x] PR has a meaningful title
- [x] Summarized changes
- [x] This PR is ready to merge and is not work in progress
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready

### CODE REVIEW

File: docs/community_snippets.md [hunk 1]
```diff
@@ -244,15 +244,15 @@ Quickly create a Pester Test for existence of a mandatory parameter by @SQLDBAWi
...
-    "Pester for Mandatory Pester": {
-        "prefix": "mandatoryParamPester",
-        "body": [
-            "It \"${1:FunctionName} Should have a mandatory parameter ${2:ParameterName}\" {",
-            "   (Get-Command ${1:FunctionName}).Parameters['${2:ParameterName}'].Attributes.Mandatory | Should -BeTrue",
-            "}"
-        ],
-        "description": "Pester Test for Parameter"
-    }
+"Pester for Mandatory Pester": {
```
Comment (rkeithhill) [inline docs/community_snippets.md]: Should this be `Pester for Mandatory Parameter`?

Comment (lipkau) [reply docs/community_snippets.md]: I guess so, based on the prefix and h3.
But I didn't change any content to the snips, as they are not mine.
(https://github.com/PowerShell/vscode-powershell/pull/1297)

Let me know if it is desired to fix that with this PR. I don't mind

**Comment by lipkau** [comment]
@tylerl0706 : I didn't do the `.json` with the snips for the module. Should I?

File: docs/community_snippets.md (hunk 1 above)
Comment (rkeithhill) [reply docs/community_snippets.md]: Yeah, that was just a typo from the original PR.  Would you mind fixing it?  Thanks.

**Comment by tylerl0706** [comment]
@lipkau if you can, that'd be great. Otherwise, it's ok

**Comment by lipkau** [comment]
Fixed the `.json` as well (although it had a lot less to fix)

**Pull request merged**